packages/js/product-editor/src/blocks/generic/linked-product-list/block.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-linked-list-field",
+	"title": "Linked product list",
```

Comment:
I wonder if it would be better to have a more generic name, such as `woocommerce/product-list-field`. The "linked" part feels unnecessary and potentially confusing.

Reply:
True, but we currently have a product list block. That was the reason why I had to pick another name.